packages/apps/graph-client/src/components/compact-transfers-table/compact-transfers-table.tsx

```diff
@@ -56,10 +56,24 @@ export const CompactTransfersTable = (
         </Table.Head>
         <Table.Body>
           {transfers.edges.map((edge, index) => {
+            /**  These transfers are going to be added to their crosschain counterpart and
+             this way we avoid repeated transfers in the table */
+            if (!chainId && edge?.node.transaction?.pactId) {
+              return <></>;
+            }
```

Comment:
To convey why this is, we could move the check to a function with a descriptive name.
Something like `isCrosschainCounterpart(edge)`?

Reply:
Sounds good! I am currently working on a PR to include item resizing and some other aspects related to the transfers page. I could add this there 😃